specification/help/resource-manager/Microsoft.Help/preview/2024-03-01-preview/help.json

```diff
@@ -0,0 +1,2568 @@
+              "$ref": "#/definitions/SimplifiedSolutionsResource"
+            }
+          },
+          "200": {
+            "description": "Create solution invocation.",
+            "schema": {
+              "$ref": "#/definitions/SimplifiedSolutionsResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Solution_Create": {
+            "$ref": "./examples/SimplifiedSolutions_Create.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "simplifiedSolutions"
+        ],
+        "description": "Get the simplified Solutions using the applicable solutionResourceName while creating the simplified Solutions.",
+        "operationId": "SimplifiedSolutions_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/SimplifiedSolutionsResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful fetched simplified Solutions result.",
+            "schema": {
+              "$ref": "#/definitions/SimplifiedSolutionsResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Solution_Get": {
+            "$ref": "./examples/SimplifiedSolutions_Get.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Help/solutions/{solutionResourceName}/warmup": {
+      "post": {
+        "tags": [
+          "solutions"
+        ],
+        "description": "Warm up the solution resource by preloading asynchronous diagnostics results into cache",
+        "operationId": "Solution_WarmUp",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/SolutionResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "solutionWarmUpRequestBody",
+            "description": "The required request body for warming up a solution resource.",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/SolutionWarmUpRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
```

Comment:
LRO-Post should use 200 or 202: https://github.com/Azure/azure-openapi-validator/blob/main/docs/post-response-codes.md